requirements.txt

```diff
@@ -0,0 +1,5 @@
+polars
+requests
+plotly
+python-dotenv
+click
```

Comment:
I've added a few dependent packages:
* Polars rather than pandas - I find it a lot easier to work with
* Click - really nice for CLI applications
* python-dotenv - allows for easy configuration via either environment variables or files

so make sure to do a `pip3 install -r requirements.txt`